webui/react/src/pages/TrialDetails/TrialRangeHyperparameters.module.scss

```diff
@@ -0,0 +1,102 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
```

Comment:
non-blocking: as discussed on yesterday's Webui AG.. There are some variables with values like this, it's unclear if we want to enforce using them, but i'm usually asked to use them by Hamid and Caleb.